### ISSUE TITLE
Fleet UI Issues #2

### ISSUE THREAD

**Issue by tfiduccia** [rancher/dashboard#1618]
- [ ] Switching workspaces while in any of the detail pages, doesn't reset the page info.  The table updates but the counts & the info under the title don't.  This happens in all the detail pages (repo, cluster, cluster group, & workspace)
![image](https://user-images.githubusercontent.com/11514927/95012265-28364c80-05ec-11eb-9f08-9f1d90a9a107.png)

- [x] When Editing a Repo, the target doesn't remember what I selected.  It just shows the type highlighted.
![image](https://user-images.githubusercontent.com/11514927/95012553-60d72580-05ee-11eb-9b36-fc7c139af127.png)

**Comment by aalves08** [comment]
Item 2 (**When Editing a Repo, the target doesn't remember what I selected. It just shows the type highlighted.**) is already fixed, I confirmed, for 2.6.4.

Item 1, indeed "nothing" happens, but, imo, we shouldn't allow the user to switch workspace (or even namespace) while on a details page. We might create unforeseen issues to our side and also it doesn't translate into a meaningful action for the user as he is looking at the details of an item that belongs to another workspace and that's just confusing. My proposal is to set the WorkspaceSwitcher as read-only (or disabled) while we are on a details page of any resource belonging to Fleet, which will not allow for changes but it will still provide context to the user.

Will move to backlog as instructed.
